siamese_network/README.md

```diff
@@ -0,0 +1,7 @@
+# Basic Siamese Network Example
+
+```bash
+pip install -r requirements.txt
+python main.py
+# CUDA_VISIBLE_DEVICES=2 python main.py  # to specify GPU id to ex. 2
+```
```

Comment:
Can you also reference the paper you're using a baseline for your implementation?

Reply:
I have referenced `FaceNet`, the closest implementation to the example's implementation. This implementation varies from FaceNet as we use the `ResNet-18` model as our feature extractor. In addition, we aren't using `TripletLoss` as the MNIST dataset is simple, so `BCELoss` can do the trick.